php/README.md

```diff
@@ -147,3 +148,17 @@ use OtherVendor\OtherPackage\Qux;
 
 // ...
 ```
+
+## Yoda Conditions
+
+You *must* use yoda conditions when doing logical comparisons `===` and `!==`. Avoid using yoda conditions for `<`, `>`, `<=` and `>=` since they are difficult to read.
```

Comment:
Although they don't say to use yoda conditions, their examples have yoda conditions. For example, in the [Symfony's coding standards](http://symfony.com/doc/current/contributing/code/standards.html) page, you can see `if ('string' === $dummy)`.